### PR TITLE
Addd Styles to the store

### PR DESCRIPTION
**Pull request by Reem-lab**
## Changes that I made here 👍 ♻️ 
 - Add `styles` directory to add styles to each component
 - Add styles to the `navbar`
 - Add styles to the `Book list`
 - Add styles to the `Form` 
 - My Final design  match Zeplin's design
 - No linters error 💯

### CODE REVIEW

File: src/styles/Books.css [hunk 1]
```diff
@@ -0,0 +1,239 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Poppins:wght@400;600;800;900&display=swap');
+
```
Comment (elyor-doniyorov) [inline src/styles/Books.css:3]: - [ ] I recommend sticking the Zeplin design as much as possible. For example, I can see the differences in spacing, colors, font size, etc. Also, I marked them with yellow color in the below screenshot 👇 
- Original 👇 
Screenshot (202)

- Yours 👇 
Screenshot (204)_LI
1. you forgot to put `bar`s between sections;
2. navbar titles should not have `underlines`; 
3. navbar `icon` which is on the right side looks different from the given design;
4. `ADD BOOK` text color should be white;
5. `Update progress` text should be as in the given design;
6. other small details should match the design;

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Reem-lab, 

Good job so far!
# To Highlight 👏 
- [ ] Linters are green ✔️ 
- [ ] PR title and commits are descriptive 👍 
- [ ] etc.

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Reem-lab,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Reem-lab** [comment]
Hi @omar-labana 
Thanks for you review 💓

**Pull request merged**